### PR TITLE
added logging for events

### PR DESCRIPTION
**Pull request by CameronRP**

### CODE REVIEW

File: eventstore/eventstore.go [hunk 1]
```diff
@@ -61,6 +62,7 @@ func Open(fileName string) (*EventStore, error) {
 // uniquely identify the event, but the contents are opaque to the
 // event store.
 func (s *EventStore) Queue(details []byte, timestamp time.Time) error {
+	log.Printf("adding new event: '%s'", string(details))
```
Comment (mjs) [inline eventstore/eventstore.go]: Does this look reasonable for a JSON blob?

Comment (CameronRP) [reply eventstore/eventstore.go]: It looks like this as a JSON blob: `adding new event2: '{"description":{"details":{"fileId":31,"volume":10},"type":"audioBait"}}'`
If I convert it to a `map[string]interface{}` with `json.Unmarshal` it looks like this: `adding new event: 'map[description:map[details:map[fileId:4 volume:10] type:audioBait]]'`
Do you want me to convert it to a `map[string]interface{}`?

Comment (mjs) [reply eventstore/eventstore.go]: Nope. As it is now is fine.

Comment (mjs) [reply eventstore/eventstore.go]: To get it to format *really* nicely with multiple lines and indenting, you'd unmarshal to a map and then marshal again with pretty formatting options turned on.

**Pull request merged**